test/quantization/core/test_workflow_module.py

```diff
@@ -1096,6 +1096,12 @@ def test_fused_mod_per_channel(self):
             torch.testing.assert_allclose(mod.state_dict()['activation_post_process.min_val'], running_min_op)
             torch.testing.assert_allclose(mod.state_dict()['activation_post_process.max_val'], running_max_op)
 
+    def test_fused_mod_reduce_range(self):
+        obs = FusedMovingAvgObsFakeQuantize(quant_min=0, quant_max=255, dtype=torch.quint8, reduce_range=True)
```

Comment:
Should the default version have reduce_range to be True, as we default to fbgemm?. Currently it looks like it does not that.